tests/functional/cypress.js

```diff
@@ -0,0 +1,12 @@
+const cypress = require('cypress');
+
+cypress.run({
+  headed: process.env.EQ_RUN_FUNCTIONAL_TESTS_HEADLESS ? false : true,
```

Comment:
Call it `headless`? it's confusing being a double negative.

Reply:
Yeah, I can't change the cypress interface unfortunately.